node/network/src/testing_utils.rs

```diff
@@ -57,8 +57,9 @@ where
     }
 }
 
-#[derive(Clone)]
+#[derive(Clone, Copy)]
```

Comment:
`Copy` is not needed? Also it is either `Copy` or `Clone`, there is no point of implementing both.

Reply:
`Clone` is needed for deriving `Copy`. `Copy` is needed in `test_ban_peer` to avoid writing `.clone()`.